rusk-wallet/src/wallet/gas.rs

```diff
@@ -10,10 +10,16 @@
 use crate::currency::Lux;
 
 /// The minimum gas limit
-pub const MIN_LIMIT: u64 = 350_000_000;
+pub const MIN_LIMIT: u64 = 50_000_000;
 
 /// The default gas limit
-pub const DEFAULT_LIMIT: u64 = 500_000_000;
+pub const DEFAULT_LIMIT: u64 = 100_000_000;
```

Comment:
with the addition of other default limits, it might make sense to call this one `DEFAULT_LIMIT_TRANSACTION` and adjust the doc accordingly

Reply:
Created issue https://github.com/dusk-network/rusk/issues/2603

Reply:
closed issue #2603 again since these changes will be handled in this PR